pages/login.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from 'react';
+import Page from '../components/Page';
+import { Form, Input, Button, Checkbox } from 'antd';
+import { MailOutlined, LockOutlined } from '@ant-design/icons';
+import axios from '../axios';
+
+const Login: FC = () => {
+
+    const onFinish = values => {
```

Comment:
Call this something else? `handleLogin` would be better

Reply:
> > 1. CSS is wonky. I recommend we use the 'styled-components' library to style our components instead of using a CSS file as it gets cluttered. Currently, we have done inline styles in JSX but we need to improve this in the future.
> 
> @PrithviAnilK Isn't it better to use styled-components when we aren't using a UI library. We already have ANTD, so what would be the point of using it here?

We can use it to style the antd components itself. Although no worries, we can use component-level CSS instead.